### ISSUE TITLE
请问为什么要代码中加入了sigmoid？

### ISSUE THREAD

**Issue by lovan114514** [xiongzihua/pytorch-YOLO-v1#45]
在net.py和resnet.py中，请问forward函数的最后为什么要加入x = torch.sigmoid(x) ？，这里之后return x应该和标签值求损失函数然后反向传播，sigmoid之后相当于归一化，明显与标签值不匹配吧

**Comment by xiazhi1** [comment]
> 在net.py和resnet.py中，请问forward函数的最后为什么要加入x = torch.sigmoid(x) ？，这里之后return x应该和标签值求损失函数然后反向传播，sigmoid之后相当于归一化，明显与标签值不匹配吧

这是为了将网络的输出约束到0-1范围内 softmax函数才是归一化
